localizations/zh-hans.json

```
@@ -0,0 +1,482 @@
+{
+    "⤡": "⤡",
+    "⊞": "⊞",
+    "×": "×",
+    "❮": "❮",
+    "❯": "❯",
+    "Loading...": "载入中...",
+    "view": "查看",
+    "api": "api",
+    "•": "•",
+    "built with gradio": "基于 Gradio 构建",
+    "Stable Diffusion checkpoint": "Stable Diffusion 模型(ckpt)",
+    "txt2img": "文生图",
+    "img2img": "图生图",
+    "Extras": "后处理",
+    "PNG Info": "PNG 信息",
+    "Image Browser": "图库浏览器",
+    "History": "历史记录",
+    "Checkpoint Merger": "模型(ckpt)合并工具",
+    "Train": "训练",
+    "Create aesthetic embedding": "生成美术风格 embedding",
+    "Settings": "设置",
+    "Prompt": "提示词",
+    "Negative prompt": "反向提示词",
+    "Run": "运行",
+    "Skip": "跳过",
+    "Interrupt": "中止",
+    "Generate": "生成",
+    "Style 1": "模版风格 1",
+    "Style 2": "模版风格 2",
+    "Label": "标签",
+    "File": "文件",
+    "Drop File Here": "拖拽文件到此",
+    "-": "-",
+    "or": "或",
+    "Click to Upload": "点击上传",
+    "Image": "图像",
+    "Check progress": "查看进度",
+    "Check progress (first)": "查看进度 (1st)",
```

```suggestion
    "Check progress (first)": "(首次)查看进度",
```

不知道这样会不会好一点，虽然根本没找到这个词条是在哪个界面出现的…

---

https://github.com/AUTOMATIC1111/stable-diffusion-webui/blob/6bd6154a92eb05c80d66df661a38f8b70cc13729/modules/ui.py#L562
it's a invisible button...